compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -83,8 +88,13 @@ def __init__(
             globus_compute_sdk.serialize.DEFAULT_STRATEGY_DATA will be used.
 
         login_manager: LoginManagerProtocol
-            Allows login logic to be overridden for specific use cases. If None, a
-            LoginManager will be used.
+            Allows login logic to be overridden for specific use cases. If None,
+            a `GlobusApp` will be used. Mutually exclusive with `app`.
+
+        app: GlobusApp
+            A `GlobusApp` that will handle authorization and storing and validating
+            tokens. If None, a standard `GlobusApp` will be used. Mutually exclusive
+            with `login_manager`.
         """
```

Comment:
- Do the backtick references properly link to their respective subjects?  Or were these intended to be double-backtick monotype?

- Consider putting `None` as a double-backtick as well.  (An improvement on what came before!)